src/App.jsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import './App.css';
 import Home from './components/Home';
-import Login from './components/Login';
+import Login from './components/auth/login/Login';
 
```

Comment:
@codeshifu why do we have to add a login folder inside of auth?

Reply:
The file structure we are following says we "group by features". I feel `authentication` is a feature, hence, the reason why I put login there. It's arguable that the `auth` folder is unnecessary. What is your take on this?